www/routes/index.tsx

```diff
@@ -228,3 +274,47 @@ function Example() {
     </section>
   );
 }
+
+function Showcase({ items }: { items: Project[] }) {
```

Comment:
It'd be nice to have some hover feedback on the showcase tiles (underline text on tile hover maybe)?

Also, can you disable tabbing to the `<a>` that surrounds the showcase image? You can already tab to the text, and if we let you tab to both it just gets confusing (because both links are the same).